addon/components/sortable-group.js

```diff
@@ -136,17 +162,140 @@ export default Component.extend({
     @method commit
   */
   commit() {
-    let items = this.get('sortedItems');
-    let groupModel = this.get('model');
-    let itemModels = items.mapBy('model');
-    let draggedItem = items.findBy('wasDropped', true);
+    const items = this.get('sortedItems');
```

Comment:
This seems to remove the concept of a `groupModel` (https://github.com/adopted-ember-addons/ember-sortable#declaring-a-group-model) and is a breaking change. Is that intentional?

Reply:
Yes I intentionally strip that out. The new 2.0.0 release will contain a lot of breaking changes (`master` already contains many), so this is going to be a major part of it.

The sole purpose of the `groupModel` was so that you have a reference to the `group` to set the new sorted array to, which I consider unnecessary as the consumer has access to the `group` without feeding it to `ember-sortable`.
```
onChange(models, draggedModel) {
 this.group.set('items', models);
}
```

My intention is to remove the bookkeeping overhead and keep the addon simple. which should be `you supply us with an array of items you want to arrange and we will give you back the new sorted order`. The added benefit of `group` seems to be trivial. However, I'm open to feedback on why maybe we should keep it.

Reply:
It's definitely a breaking change; I haven't used this addon in quite awhile, so I don't if it is a big deal or not. That said, it seems like this PR should make some changes to the README and other documentation.

Reply:
I plan on update all the README and documentation when all the preparation for the next release is ready. We're not planning to release anything until the first major release is ready.

Reply:
Imo, I think changing the README now might confuse people to think that the current release (which is way behind) contain these changes when they come and look at the repo.

Reply:
I plan to create a `migration doc`, update `readme` and other documentations when all the implementations are completed prior to the next release. Since right now we might still need to re-iterate and have the overhead of keep updating the docs.

Reply:
@kategengler, actually I added the `groupModel` support...It might be useful for future feature requests that I'm getting. But regarding README and other documentation, are you ok with me holding off on that until after the migration to v2 is done?

Reply:
👋 I'm not a maintainer of this repo, was just making a suggestion, earlier 😀

Reply:
Ah ok, appreciate your feedback!